src/payment/strategies/credit-card/credit-card-payment-strategy.ts

```diff
@@ -98,4 +104,8 @@ export default class CreditCardPaymentStrategy implements PaymentStrategy {
 
         return isHostedPaymentFormEnabled;
     }
+
+    private _ishostedFieldAvailable(options?: PaymentInitializeOptions): boolean {
+        return !(values(options && options.creditCard && options.creditCard.form.fields).every(isUndefined));
```

Comment:
Should probably consider `null` as well. Perhaps `isNil` is more suitable as it covers both `undefined` and `null`.